Added automation example for qrcode integration (#17677)
home-assistant/home-assistant.github.io: source/_integrations/qrcode.markdown

      required: false
      type: string
{% endconfiguration %}

## Basic example

An automation using this integration could look like this:

```yaml
automation:

>   - alias: 'Catch QR code and arm the alarm system'

Reviewer: ```suggestion
  - alias: "Catch QR code and arm the alarm system"
```

    trigger:

>       platform: state

Reviewer: This needs to be a list, see our YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      - platform: state
```

      entity_id: image_processing.qr_front_door

>       to: !secret ARM_QR_CODE_VALUE

Reviewer: Please don't use secrets in examples

    action:

>       service: alarm_control_panel.alarm_arm_away

Reviewer: This needs to be a list, see our YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      - service: alarm_control_panel.alarm_arm_away
```

>       entity_id: alarm_control_panel.home_alarm

Reviewer: Please use service targets, see our style guide for more information: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      target:
        entity_id: alarm_control_panel.home_alarm
```

      data:

>         code: !secret MY_ALARM_CODE

Reviewer: Please don't use secrets in these examples
